descriptor.md

```diff
@@ -220,3 +232,5 @@ In the following example, the descriptor indicates the type of artifact it is re
 [rfc7230-s2.7]: https://tools.ietf.org/html/rfc7230#section-2.7
 [sha256-vs-sha512]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/hsMw7cAwrZE
 [iana]: https://www.iana.org/assignments/media-types/media-types.xhtml
+[blake3]: https://github.com/C2SP/C2SP/blob/BLAKE3/v1.0.0/BLAKE3.md
```

Comment:
Is there an RFC we can point to? Also I'm not sure if we can articulate the difference between  https://www.ietf.org/archive/id/draft-aumasson-blake3-00.html and a GitHub link above.

Reply:
As per the commentary, RFC was submitted for process sake, but deemed ineffective/unnecessary. So this repo may be source of truth afaik.